README.md

```diff
@@ -4,101 +4,66 @@ Core database, replication, and chat APIs for cabal.
 
 ## Usage
 
-    npm install cabal-core
+    npm install --save cabal-core
 
 ## API
 
-> var Cabal = require('cabal-node')
+<!-- Generated by documentation.js. Update this documentation by updating the source code. -->
```

Comment:
sorry could you remove this auto-generated documentation? looking at the diff I'm seeing useful information being removed from the readme. maybe we could discuss the idea of auto-docs in another PR?

Reply:
I was just running the `npm run doc` command. I'll update the `README.md` from master then run it again and see if this fixes it. Otherwise, I might need a pointer if I'm still stuck. Thanks!